app/src/main/java/com/github/gotify/service/WebSocketService.java

```diff
@@ -314,6 +315,19 @@ private void showNotification(
         b.setContentText(message);
         b.setStyle(new NotificationCompat.BigTextStyle().bigText(formattedMessage));
 
+        String notificationImageUrl =
+                Extras.getNestedValue(String.class, extras, "client::notification", "imageUrl");
+
+        if (notificationImageUrl != null) {
+            try {
+                b.setStyle(
+                        new NotificationCompat.BigPictureStyle()
+                                .bigPicture(picassoHandler.getImageFromUrl(notificationImageUrl)));
+            } catch (IOException e) {
+                Log.e(e.toString());
+            }
```

Comment:
I changed this to 
```
            } catch (Exception e) {
                Log.e("Error loading bigImageUrl", e);
            }
```
because the user could supply a malformed url that would crash the websocket service. also it is better to just pass the exception directly to the logged then the stacktrace is inside the logs.